src/django_pg_migration_tools/management/commands/migrate_with_timeouts.py

```diff
@@ -69,3 +61,13 @@ def _migrate(self, *django_migrate_args: Any, **django_migrate_kwargs: Any) -> N
             statement_timeout=self.statement_timeout,
         ):
             super().handle(*django_migrate_args, **django_migrate_kwargs)
+
+
+def _parse_non_required_ms_option_to_timedelta(
+    options: Any, key: str
+) -> datetime.timedelta | None:
+    value = options.pop(key, None)
+    if value is None:
+        return None
+    assert isinstance(value, int)
+    return datetime.timedelta(seconds=int(value / 1_000))
```

Comment:
Why have the argument in milliseconds if it's going to be rounded down to a whole number of seconds anyway?

Reply:
[Addressed] That's a problem. I wasn't aware that `timedelta` had a `miliseconds` argument as I was biased by the author's preference for microseconds:

![image](https://github.com/user-attachments/assets/3c828c26-9841-444f-93b3-f130576d234d)